ci_scripts/build_android_debug.sh

```diff
@@ -17,7 +17,8 @@ cmake . -G Ninja -B "build-android-$ANDROID_ABI-debug" \
 -DANDROID_NDK="$ANDROID_NDK" \
 -DCMAKE_TOOLCHAIN_FILE="$ANDROID_NDK/build/cmake/android.toolchain.cmake" \
 -DCMAKE_BUILD_TYPE=Debug \
--DBASISU_SUPPORT_SSE=OFF
+-DBASISU_SUPPORT_SSE=OFF \
+-DISA_NONE=ON
```

Comment:
Same as comment at same place in build_android.sh.

Reply:
Replied to that comment. TL,DR. Its there so one can build all SIMD options.